### DISCUSSION TITLE
关于该仓库重构成为真正的monorepo的一些思考？

### DISCUSSION THREAD

**Discussion by kimmy-wang** [element-plus/element-plus#8243] [General]
@sxzz 现在打包的时候会打包在dist/element-plus目录下，建议分开打包。

**Comment by sxzz** [comment]
是的，我们将在 Element Plus 3 中改善这一点。我已经做了一个原形项目，已实现你所说的模式。
https://github.com/sxzz/element-plus-next

**Reply by kimmy-wang** [reply]
3.0啥时候发布呢？

**Reply by sxzz** [reply]
https://twitter.com/sanxiaozhizi/status/1542773112877764608
